scrapy/extensions/feedexport.py

```diff
@@ -241,28 +241,19 @@ def __init__(self, crawler):
 
         self.storages = self._load_components('FEED_STORAGES')
         self.exporters = self._load_components('FEED_EXPORTERS')
+        self.storage_batch = self.settings.getint('FEED_STORAGE_BATCH')
```

Comment:
Maybe `FEED_STORAGE_BATCH_SIZE` is a better name.
Also, we could consider time based deliveries (say every X minutes), this one is just for consideration.